endpoint/data_backends/sqlite/manage.py

```diff
@@ -0,0 +1,88 @@
+from model import User, Group, Token
+from endpoint.lib import crypto
+from database import db
+
+
+def init_db():
+    db.create_all()
+
+
+def get_user_by_username(username):
+    user = User.query.filter(User.username == username).first()
+    if not user:
+        return False
```

Comment:
What do you think of this?

What is object returned by the query? Right now I am thinking of how returning different types (user and Boolean) could be confusing. Can you return ‘None’??

Reply:
Good point, I was thinking about this too and how we deal with this at my day job. What we do is that we have a custom but very thin `Results` object wrapper that has a success, data and errors attribute. If the result is successful then `results.success` is set to `True` and the `results.data` is set to a dict that contains everything you want to return. If the call fails for some reason then `results.success` is set to `False` and `results.errors` is set to a list of exceptions or text string representations of the errors. Do you think I should do that here? Totally could and would be a nice consistent way to move forward.